### ISSUE TITLE
Prevent previous versions of WP-Librarian from activating

### ISSUE THREAD

**Issue by kittsville** [kittsville/WP-Librarian#61]
Currently users can remove their current version of WP-Librarian and install a previous version. Considering WP-Librarian makes various changes to how it stores/handles data with each Alpha/Beta release this can result in things going seriously wrong. The worst case being a previous version of WP-Librarian overwriting the version in the database. This would mean, if the latest version were re-installed the migrations would be performed a second time.

tl;dr stop people from corrupting their library

**Comment by kittsville** [comment]
I might hotfix this to all previous releases, depending on time/if I think pre-releases really need hotfixing.

**Comment by kittsville** [comment]
Now would also be a good time to add PHP version restrictions
